modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/util/setDataRecord.es.js

```diff
@@ -36,7 +36,7 @@ export default (
 		return;
 	}
 
-	let _value = value;
+	let _value = JSON.stringify(value);
```

Comment:
Hi, @victorg1991. I saw you comment over my review before. Thanks for bringing that to my attention. That change was actually something a few members of the team discussed together. Before, we were sending objects over requests, but we believed to be unnecessary, since the values needed to be Strings after all in order to create the DDMFormValues inside the DefaultMapToDDMFormValuesConverterStrategy. So we removed that part in an attempt of doing that, believing that all values were already Strings. Your fix actually brought to our attention that this wasn't the case. So I believe this is the piece that was missing from that pull request you shared with me.
Thanks a lot for your contribution.